symengine/tests/basic/test_arit.cpp

```diff
@@ -987,6 +987,36 @@ TEST_CASE("Pow: arit", "[arit]")
     r1 = pow(mul(sqrt(mul(y, x)), x), i2);
     r2 = mul(pow(x, i3), y);
     REQUIRE(eq(*r1, *r2));
+
+    r1 = exp(mul(I, x));
+    r2 = pow(E, mul(I, x));
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = exp(mul(I, pi));
+    r2 = pow(E, mul(I, pi));
+    REQUIRE(eq(*r1, *r2));
+
+    r1 = exp(mul(I, pi));
+    REQUIRE(eq(*r1, *minus_one));
+
+    r1 = exp(mul(mul(I, minus_one), pi));
+    REQUIRE(eq(*r1, *minus_one));
+
+    r1 = exp(div(mul(I, pi), integer(2)));
+    REQUIRE(eq(*r1, *I));
+
+    r1 = exp(mul(div(mul(I, pi), integer(2)), minus_one));
+    REQUIRE(eq(*r1, *mul(I, minus_one)));
+
+    r1 = div(exp(mul(mul(I, pi), x)), exp(x));
+    REQUIRE(r1->__str__() == "exp(-x + I*x*pi)");
```

Comment:
Can you change this test to checking that `r1` is a `Pow` and that the base and exp are `E` and `-x + I * x * pi` respectively ?